lib/util/middlewares.js

```diff
@@ -187,10 +187,30 @@ exports.loginRequired = function(crowi, app) {
   };
 };
 
+/**
+ * Extract Bearer token from Authorization header.
+ *
+ * @param {Object} headers HTTP request headers
+ * @param {string} headers.authorization Authorization header value.
+ * @return {?string} found access_token or null.
+ */
+var extractBearerToken = (headers) => {
+  const v = headers.authorization;
```

Comment:
Comment: here useing `const`, so `var` of `var extractBearerToken` also should be `const`?